tools/topology/topology2/cavs-nocodec.conf

```diff
@@ -94,8 +95,12 @@ IncludeByKey.NUM_DMICS {
 }
 
 # include deep buffer config if buffer size is in 1 - 1000 ms.
-IncludeByKey.DEEPBUFFER_FW_DMA_MS{
-        "[1-1000]" "platform/intel/deep-buffer.conf"
+IncludeByKey.PASSTHROUGH {
+"false" {
```

Comment:
indentation?

Reply:
This was intentional a bit like switch/case where they're at the same indentation

Reply:
it's inconsistent with other uses of IncludeByKey, see e.g. the "[1-1000]" that is removed, it wasn't indented.